swap/swap_test.go

```diff
@@ -118,15 +118,7 @@ func newTestBackend(t *testing.T) *swapTestBackend {
 
 	// deploy the ERC20-contract
 	// ignore receipt because if there is no error, we can assume everything is fine on a simulated backend
-	tokenAddress, _, token, err := contractFactory.DeployERC20Mintable(bind.NewKeyedTransactor(ownerKey), defaultBackend)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defaultBackend.Commit()
```

Comment:
Why don't we need the `Commit` anymore?

Reply:
It's still there. It just looks like it was removed in the diff. (because it thinks the `defaultBackend.Commit()` that is there now was from the lines below the `token.Mint` call).